code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -61,84 +62,88 @@
+			new /obj/item/gun/energy/recharge/ebow(src) // 10 tc
+			new /obj/item/pen/sleepy(src) // 4 tc
+			new /obj/item/healthanalyzer/rad_laser(src) // 3 tc
+			new /obj/item/chameleon(src) // 7 tc
+			new /obj/item/soap/syndie(src) // 1 tc
+			new /obj/item/clothing/glasses/thermal/syndi(src) // 4 tc
+			new /obj/item/flashlight/emp(src) // 2 tc
+			new /obj/item/jammer(src) // 5 tc
 
 		if(KIT_GUN)
-			new /obj/item/gun/ballistic/revolver(src)
+			new /obj/item/gun/ballistic/revolver(src) // 13 tc
+			new /obj/item/ammo_box/a357(src) // 4 tc
 			new /obj/item/ammo_box/a357(src)
-			new /obj/item/ammo_box/a357(src)
-			new /obj/item/card/emag(src)
-			new /obj/item/grenade/c4(src)
-			new /obj/item/clothing/gloves/color/latex/nitrile(src)
+			new /obj/item/storage/belt/holster/chameleon(src) // 1 tc
+			new /obj/item/card/emag/doorjack(src) // 3 tc replaced the emag with the doorjack
+			new /obj/item/grenade/c4(src) // 1 tc
+			new /obj/item/clothing/gloves/color/latex/nitrile(src) // ~1 tc for whole outfit
 			new /obj/item/clothing/mask/gas/clown_hat(src)
 			new /obj/item/clothing/under/suit/black_really(src)
 			new /obj/item/clothing/neck/tie/red/hitman(src)
 
 		if(KIT_SCREWED)
-			new /obj/item/sbeacondrop/bomb(src)
-			new /obj/item/grenade/syndieminibomb(src)
-			new /obj/item/sbeacondrop/powersink(src)
-			new /obj/item/clothing/suit/space/syndicate/black/red(src)
+			new /obj/item/sbeacondrop/bomb(src) // 11 tc
+			new /obj/item/grenade/syndieminibomb(src) // 6 tc
+			new /obj/item/sbeacondrop/powersink(src) // 11 tc
+			new /obj/item/clothing/suit/space/syndicate/black/red(src) // outfit 1 tc
 			new /obj/item/clothing/head/helmet/space/syndicate/black/red(src)
-			new /obj/item/encryptionkey/syndicate(src)
+			new /obj/item/encryptionkey/syndicate(src) // 2 tc
 
 		if(KIT_MURDER)
-			new /obj/item/melee/energy/sword/saber(src)
-			new /obj/item/clothing/glasses/thermal/syndi(src)
-			new /obj/item/card/emag(src)
-			new /obj/item/clothing/shoes/chameleon/noslip(src)
-			new /obj/item/encryptionkey/syndicate(src)
-			new /obj/item/grenade/syndieminibomb(src)
+			new /obj/item/melee/energy/sword/saber(src) // 8 tc
+			new /obj/item/clothing/glasses/thermal/syndi(src) // 4 tc
+			new /obj/item/card/emag/doorjack(src) // 3 tc
+			new /obj/item/clothing/shoes/chameleon/noslip(src) // 2 tc
+			new /obj/item/encryptionkey/syndicate(src) // 2 tc
+			new /obj/item/grenade/syndieminibomb(src) // 6 tc
 
 		if(KIT_IMPLANTS)
-			new /obj/item/implanter/freedom(src)
-			new /obj/item/implanter/uplink/precharged(src)
-			new /obj/item/implanter/emp(src)
-			new /obj/item/implanter/explosive(src)
-			new /obj/item/implanter/storage(src)
+			new /obj/item/implanter/freedom(src) // 5 tc
+			new /obj/item/implanter/uplink/precharged(src) // 4 tc
+			new /obj/item/implanter/emp(src) // 1 tc
+			new /obj/item/implanter/explosive(src) // 2 tc
+			new /obj/item/implanter/storage(src) // 8 tc
+			new /obj/item/implanter/radio/syndicate(src) // 4 tc
```

Comment:
Gonna be honest it doesn't need the extra implants

Reply:
oh i just misunderstood what this one did. that actually makes it kinda boring to me but youre right it doesnt need the extra ones if it just has 10 TC

Reply:
the value is that it comes with a whole second uplink worth of TC